homework4/Routers/Routers/Edge.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="Edge.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Routers;
+
+/// <summary>
+/// this edge.
+/// </summary>
+public class Edge
```

Comment:
Это могло бы быть record-ом:
```suggestion
public record Edge(int FirstVertex, int SecondVertex, int Bandwidth);
```
И всё, весь класс в одну строчку :)